frontstage_api/controllers/collection_exercise_controller.py

```diff
@@ -37,3 +38,18 @@ def get_collection_exercise_events(collection_exercise_id):
 
     logger.debug('Successfully retrieved collection exercise events', collection_exercise_id=collection_exercise_id)
     return json.loads(response.text)
+
+
+def get_collection_exercise_event(collection_exercise_id, tag):
+    logger.debug('Retrieving collection exercise event', collection_exercise_id=collection_exercise_id, tag=tag)
+    url = app.config['RM_COLLECTION_EXERCISE_EVENT'].format(collection_exercise_id, tag)
+
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        raise ApiError(url=url, status_code=response.status_code,
+                       description='Failed to retrieve collection exercise event',
+                       collection_exercise_id=collection_exercise_id, tag=tag)
+
+    logger.debug('Successfully retrieved collection exercise event', collection_exercise_id=collection_exercise_id, tag=tag)
+    return json.loads(response.text)
```

Comment:
response.json() ?

Reply:
The function above `def get_collection_exercise_events(collection_exercise_id):` uses 
`return json.loads(response.text)` so would be nice to have both the same

Reply:
In general we should be using `response.json()`  it's just cleaner. I didn't know about it for ages so we use the other way across the frontstages/backstages 🤦‍♂️.  I'll update these both